extensions/functions_arithmetic.yaml

```diff
@@ -497,3 +536,6 @@ window_functions:
         decomposable: NONE
         return: i64?
         window_type: PARTITION
+        decomposable: MANY
+        intermediate: "fp64?"
+        return: fp64?
```

Comment:
This doesn't look right.

Reply:
Typo, fixed.

Reply:
No, what doesn't look right is that these three lines came out of nowhere at the bottom of the file. They should be removed.

Reply:
Yes, I didn't catch that sorry. Now it should be okay.